src/main/java/core/basesyntax/service/FruitService.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.AbstractOperation;
+import core.basesyntax.model.OperationFactory;
+import core.basesyntax.model.OperationSet;
+import core.basesyntax.model.entities.Fruit;
+import core.basesyntax.service.io.ReportWriter;
+import java.util.List;
+
+public class FruitService extends AbstractService<Fruit> {
+
+    public void importData(List<String[]> data) {
+        OperationFactory<Fruit> operationFactory = new OperationFactory<>(warehouse);
+        Validator<Fruit> validator = new Validator<>(warehouse);
```

Comment:
let's better inject these dependencies as fields into the constructor of FruitService in main() method

Reply:
Why?

Reply:
this is the good practice for injecting dependencies(other services, for example) if you use them in your class
they should be fields, not local variables - no code repetition is needed if you wish to use the same dependency several times in your code